helpers/appHelper.js

```diff
@@ -0,0 +1,14 @@
+function findWinner(pokemons, berries) {
```

Comment:
the name of the function doesn't reflect its content. the function not only finds the winner, it also shows information about all pokemons, feeds them. I would say that it is better not to combine all these actions under a single function